tutorials/shading/shading_reference/shading_language.rst

```diff
@@ -477,7 +477,7 @@ You can set uniforms in the editor in the material. Or you can set them through
           must match *exactly* to the name of the uniform in the shader or else it will not be recognized.
 
 Any GLSL type except for *void* can be a uniform. Additionally, Godot provides optional shader hints
-to make the compiler understand for what the uniform is used.
+to make the compiler understand for what the uniform is used. ``hint_albedo`` is required for proper sRGB->linear conversion, as Godot’s 3D engine renders in linear color space.
```

Comment:
Maybe just add "For example..." Otherwise this sentence feels really out of context.